### ISSUE TITLE
ebay-textbox: floating label generates wrong HTML

### ISSUE THREAD

**Issue by darkwebdev** [eBay/ebayui-core#1027]
# Bug Report

## Description
When using floating label generated HTML is different from what I see in Skin 9/DS6 documentation.

Usage:
```html
<ebay-textbox floating-label="First Name" />
```

Expected:
```html
<span class="floating-label">
    <label class="floating-label__label" for="firstName">First Name</label>
    <span class="textbox">
        <input class="textbox__control textbox__control--underline" id="firstName" type="text" />
    </span>
</span>
```

Generated:
```html
<span class="textbox">
	<label for="dynamic-id" class="floating-label__label floating-label__label--inline">First Name</label>
	<input id="dynamic-id" class="textbox__control textbox__control--underline" type="text">
</span>
```

**Comment by ianmcburnie** [comment]
Seems to be working somehow, but yes floating label class should be a wrapper around a textbox (like in the skin example).

**Comment by ianmcburnie** [comment]
Fixed in 10.6.0 branch.
